pkg/commands.go

```diff
@@ -1,4 +1,4 @@
-package common
+package pkg
```

Comment:
I think this can be moved into the constant package.

Also the comment originally put is not very good.

These are used in cctxs of type cmd to give an instruction to execute on the external chain (sending a whitelist erc20 tx from tss for example)

```
// CmdWhitelistERC20 is used for CCTX of type cmd to give the instruction to the TSS to whitelist an ERC20 on an exeternal chain
// CmdMigrateTssFunds is used for CCTX of type cmd to give the instruction to the TSS to transfer its funds on a new address
```